src/campsite-info/components/camp-description/camp-description.component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { descriptionStyles } from './camp-description.styles';
+
+export const CampDescription = () => {
+  return (
+    <Box sx={descriptionStyles} component="body">
```

Comment:
The body tag is meant to enclose the whole of the presentational part of your website and should not be used here.